src/Events.Functions/Clients/EventsClient.cs

```diff
@@ -110,7 +110,7 @@ public async Task PostOutbound(CloudEvent cloudEvent)
         {
             StringContent httpContent = new(JsonSerializer.Serialize(cloudEvent), Encoding.UTF8, "application/json");
 
-            string endpointUrl = "push";
+            string endpointUrl = "outbound";
```

Comment:
This should be handled in #178. 
Suggesting we create this as a seperate PR so the current PR is isolated to the Events web api, rather than the function as well

Reply:
Agreed.